src/hotspot/share/opto/type.cpp

```diff
@@ -3133,7 +3133,7 @@ const TypeRawPtr *TypeRawPtr::make( enum PTR ptr ) {
 }
 
 const TypeRawPtr *TypeRawPtr::make( address bits ) {
-  assert( bits, "Use TypePtr for null" );
+  assert( bits != nullptr, "Use TypePtr for null" );
```

Comment:
I'm not fond of those spaces, but they follow the style used throughout this file.

Reply:
Although it looks like only 1/3 of the asserts in this file have extra whitespace, including the one being touched
here.  So sure, I can remove the extraneous whitespace from this function, since touching it anyway.